bigquery/google/cloud/bigquery/table.py

```diff
@@ -387,10 +396,10 @@ def schema(self):
     def schema(self, value):
         if value is None:
             self._properties["schema"] = None
-        elif not all(isinstance(field, SchemaField) for field in value):
-            raise ValueError("Schema items must be fields")
-        else:
-            self._properties["schema"] = {"fields": _build_schema_resource(value)}
+            return
```

Comment:
Clearer as an `else:`?

Reply:
On a second thought, yes, the current construct indeed looks slightly odd.